### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by alchemiker** [ltex-plus/vscode-ltex-plus#21] [Show and tell]
Hey, I just wanted to say thank you for this fork + fix, you really saved me and my thesis that I have to write 😁

**Comment by ajinkya-k** [comment]
Came here to say the same! Thanks for the successor/fork.

**Comment by demisjohn** [comment]
I can't believe how easy that was to install in VSCode, and within minutes have offline spellchecking for Typst.  Very impressive!
